resources/tests/YulTests/EmptyContract.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+cd ../../../
+
+sbt "runMain edu.cmu.cs.obsidian.Main --yul resources/tests/YulTests/EmptyContract.obs"
+
+cd EmptyContract
+
+solc --strict-assembly EmptyContract.yul
```

Comment:
Don't you need to check to make sure this succeeded?

Reply:
oh yeah my bad. Adding a if statement to check if $? is zero.